### ISSUE TITLE
Error 404 on Webserver with Quilt Modloader

### ISSUE THREAD

**Issue by rdvdev2** [Minecraft-Transit-Railway/Minecraft-Transit-Railway#726]
**Expected Behaviour**
The transit map appears

**Actual Behaviour**
Error 404

**Reproduction Steps**
Steps to reproduce the behaviour:
1. Start the server with the mod
2. Open IP:8888/index.html

**Screenshots**
![imatge](https://github.com/jonafanho/Minecraft-Transit-Railway/assets/33713918/3a2576c2-db01-4bd9-b8ad-97b2ca4129a3)

**Environment**
- Minecraft Version: 1.19.2
- Installed Mods: [Modlist](https://github.com/rdvdev2/server_del_grupito/tree/master/mods)

**Additional Information**
The server is running on docker with this [image](https://github.com/itzg/docker-minecraft-server) and behind a VPN. TCP port 8888 is opened through the VPN, firewall and docker config.

**Comment by Ultimate-USA-YT** [comment]
http://localhost:8888/index.html

**Comment by kettle-7** [comment]
they already specified that they are going to this link

**Comment by Kenny-Hui** [comment]
Reproducible on Quilt Loader 0.23.1, the online system map cannot be accessed and only works on Fabric and Forge.
```
[02:40:16] [Server thread/INFO]: Started m.l.o.e.j.s.ServletContextHandler@306c527a{/,null,AVAILABLE}
```

**Comment by jonafanho** [comment]
This should be fixed in 4.0.0. Please reopen this bug if you still encounter this issue with 4.0.0, thanks!
